lib/reporters/cli/index.js

```diff
@@ -57,6 +58,9 @@ extractSNR = function (executions) {
  * @param {Boolean=} reporterOptions.noFailures - Boolean flag to turn off failure reporting altogether, if set to true.
  * @param {Boolean=} reporterOptions.noConsole - Boolean flag to turn off console logging, if set to true.
  * @param {Boolean=} reporterOptions.noBanner - Boolean flag to turn off newman banner, if set to true.
+ * @param {Boolean=} reporterOptions.showTimestamp - Boolean flag to log timestamp of each request, if set to true.
+ * @param {Boolean=} reporterOptions.showTimestampNoTz - Boolean flag to log timestamp of each request, if set to true.
```

Comment:
no sure if this is still needed.

Reply:
I added this to maintain consistency with the rest of the JSDoc comment.

Reply:
Although being the CLI reporter maybe it's okay to not have it. I'm open to both options as it does not affect the usage either way.

Reply:
I meant having multiple options/formats for logging timestamps.